### ISSUE TITLE
Better text editing within Neos

### ISSUE THREAD

**Issue by sirkitree** [Frooxius/NeosPublic#136]
## Problem/Motivation
It'd be great to have better text editing capabilities.

Currently you can
1. Create a text object and type a small amount of text.
1. Import a text file, and edit it.

However, in both cases, editing a large amount of text is a bit tedious. The text ends up showing up in the inspector window as unreadable when there is a large amount.

## Proposed Solution

In both cases having an easier means of placing the cursor is desirable. I know of the value you can modify within the inspector for this purpose, or you can use the arrow keys on the keyboard, but this is tedious when there is a lot of text. Being able to point and click where you want it would make this much easier, such as you can do with a mouse currently. When the laser points to the text, it could change color or shape to indicate that you're pointer will place the cursor, and then you can use the keyboard as normal.

I'd also love to eventually see some basic formatting options, such as **bold** and *italic*, but maybe that is better left as another issue.

**Comment by Frooxius** [comment]
Yes this one is planned too, but I need to add tighter integration with the text rendering system, right now I don't have a way of checking where in the text has the user clicked. I'll also add a pop-up dialog for editing a text field if you want to do a multiline text.

You can do basic formatting right now using tags. E.g. typing <b>This is bold</b> and <i>This is italic</i> into the text field should work (if the rich text is enabled).

**Comment by Aerizeon** [comment]
I would definitely like to see MultiLine mode implemented - it would be very useful for longer text - especially if it means we can use the return/enter key as newline instead of losing the control's focus (and needing to do shift + enter)
